bin/CIConfigGen.php

```diff
@@ -1,10 +1,9 @@
+#!/usr/bin/env php
 <?php
 
-namespace CIConfigGen;
```

Comment:
No need for this in bin files, as they're unique entry point without any classes